### PR TITLE
INTG-1544: create schema only command

### PR DESCRIPTION
**Pull request by atashbahar**
a new flag `--create-schema-only` which allows users to create the schemas only.

### CODE REVIEW

File: cmd/iauditor-exporter/cmd/export/export.go [hunk 1]
```diff
@@ -59,22 +59,36 @@ func getAPIClient() api.APIClient {
...
 
 	exportPath := viper.GetString("export.path")
 	os.MkdirAll(exportPath, os.ModePerm)
 
 	exporter, err := feed.NewCSVExporter(exportPath)
 	util.Check(err, "unable to create exporter")
 
+	schemaOnly := viper.GetBool("export.schema_only")
+
+	if schemaOnly {
```
Comment (BCook98) [inline cmd/iauditor-exporter/cmd/export/export.go:87]:
```suggestion
	if viper.GetBool("export.schema_only") {
```

File: cmd/iauditor-exporter/cmd/export/export.go [hunk 2]
```diff
@@ -59,22 +59,36 @@ func getAPIClient() api.APIClient {
...
 
 func runSQL(cmd *cobra.Command, args []string) error {
-	apiClient := getAPIClient()
 
 	exporter, err := feed.NewSQLExporter(viper.GetString("db.dialect"), viper.GetString("db.connection_string"), true)
 	util.Check(err, "unable to create exporter")
 
+	schemaOnly := viper.GetBool("export.schema_only")
+
+	if schemaOnly {
```
Comment (BCook98) [inline cmd/iauditor-exporter/cmd/export/export.go:68]:
```suggestion
	if viper.GetBool("export.schema_only") {
```

File: internal/app/feed/export_feeds.go [hunk 3]
```diff
@@ -9,6 +9,33 @@ import (
 	"github.com/spf13/viper"
 )
 
+func CreateSchemas(exporter Exporter) error {
+	logger := util.GetLogger()
+	logger.Info("Creating schemas started")
+
+	feeds := []Feed{
```
Comment (BCook98) [inline internal/app/feed/export_feeds.go:16]: Nice! We should be able to apply this to Export feeds as well. Would be a lot nicer...

**Review by BCook98 — APPROVED** [review]
Nice work!

Only a couple of minor comments

File: internal/app/feed/export_feeds.go (hunk 3 above)
Comment (atashbahar) [reply internal/app/feed/export_feeds.go:16]: I think we can create something like this and iterate over it: 
```
feeds := map[string]Feed {
   "inspections": &InspectionFeed{...},
   ...
}
```

Comment (BCook98) [reply internal/app/feed/export_feeds.go:16]: Perhaps, the `Feed` also has a `Name()` method which you can use to compare agains the enabled map.

Comment (atashbahar) [reply internal/app/feed/export_feeds.go:16]: You're right, we can use the `Name()` instead. Do you want me to make the change or should we do it in a separate PR?

Comment (BCook98) [reply internal/app/feed/export_feeds.go:16]: Let's do it separately. No need to bring it into this PR.

**Review by BCook98 — APPROVED** [review]

**Pull request merged**